crates/viewer/re_view_spatial/src/visualizers/video/mod.rs

```diff
@@ -102,6 +113,17 @@ fn show_video_error(
     video_size: glam::Vec2,
     entity_path: &EntityPath,
 ) {
+    // Register the full video bounds regardless for more stable default view extents for when the error
+    // goes in and out of existance.
+    // The size of the error icon depends on the bounds in turn, making this extra important!
```

Comment:
Does it have to though? Why not always have the error have the same size as the video (if known), or a fixed size if not known?

Reply:
I think we used to do that a while ago but one of us two decided that it was nicer for varying 2d view sizes, see comment a but further below

```rs
    // If we're in a 2D view, make the error rect take a fixed amount of view space.
    // This makes it look a lot nicer for very small & very large videos.
    if let Some(state) = ctx.view_state.as_any().downcast_ref::<SpatialViewState>() {
        if let Some(bounds) = state.visual_bounds_2d {
            // Aim for 1/8 of the larger visual bounds axis.
```

Reply:
either way "registering" the full bounds here is a good idea because the video may show up eventually :)

Reply:
talked about it: problem with the current approach is also that icons may cover other videos in the same view. bad!